index.html

```diff
@@ -0,0 +1,309 @@
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                </ul>
+            </div>
+
+            <div class="categories">
+                <div class="categories-heading-box">
+                    <img src="img/skills-icon.svg" alt="round icon" />
+                    <h3 class="categories-heading">Skills</h3>
+                </div>
+                <ul class="categories-list">
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+            </div>
+        </div>
+    </section>
+
+    <footer class="contact">
+        <div class="contact-box">
+            <div class="contact-text-box padding-left-right">
+                <h2 class="contact-heading">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </h2>
+            </div>
+            <form class="contact-form padding-left-right" action="https://formspree.io/f/xayvglaa" method="post">
+                <fieldset class="contact-group">
+                    <input type="text" name="user_full_name" placeholder="Full name" required maxlength="30" />
+                    <input type="email" name="user_email" placeholder="Email address" required />
+                    <textarea name="user_message" placeholder="Write me something..." required maxlength="500"></textarea>
+                </fieldset>
+                <input class="btn-form" type="submit" name="submit" value="Get in touch" />
+            </form>
+        </div>
+
+        <div class="footer-social-icons">
+            <ul class="social-icons-list">
+                <li>
+                    <a href="#">
+                        <img class="icon" src="img/icon-gitHub.svg" alt="github social icon" />
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#">
+                        <img class="icon" src="img/icon-linkedin.svg" alt="linkedin social icon" />
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#">
+                        <img class="icon" src="img/icon-angelist.svg" alt="pinterest angelist icon" />
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#">
+                        <img class="icon" src="img/icon-twitter.svg" alt="twitter social icon" />
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#">
+                        <img class="icon" src="img/icon-medium.svg" alt="mercedes social icon" />
+                    </a>
+                </li>
+            </ul>
+        </div>
```

Comment:
- [ ] Lastly, the `social icons ` in the footer section are also currently not showing. So kindly fix that as well :100: 
- Your Design
![image](https://github.com/Tarto-4/Desktop_Portfolio/assets/102699958/5183fd96-746d-4f3c-913c-8d854919b184)

- The Figma Design
![image](https://github.com/Tarto-4/Desktop_Portfolio/assets/102699958/ca467b57-aff1-4cf4-97a0-9b15405374ce)